frontend/appflowy_flutter/lib/ai/service/ai_prompt_input_bloc.dart

```diff
@@ -18,49 +12,31 @@ part 'ai_prompt_input_bloc.freezed.dart';
 
 class AIPromptInputBloc extends Bloc<AIPromptInputEvent, AIPromptInputState> {
   AIPromptInputBloc({
+    required String objectId,
     required PredefinedFormat? predefinedFormat,
-  })  : _listener = LocalAIStateListener(),
-        super(AIPromptInputState.initial(predefinedFormat)) {
+  })  : _aiModelStateNotifier = AIModelStateNotifier(objectId: objectId),
+        super(AIPromptInputState.initial(objectId, predefinedFormat)) {
```

Comment:
Can we use the same `AIModelStateNotifier` for the `AIPromptInputBloc` and the `SelectModelBloc`? That way we can speed up loading the models when opening the popup.

Reply:
We can do that, but it adds complexity in managing init, startListening, and stop. If speed is a concern, we can optimize further.